ts/features/idpay/payment/components/Barcode/useIOBarcodeFileReader.tsx

```diff
@@ -1,8 +1,8 @@
 import * as A from "fp-ts/lib/Array";
 import * as E from "fp-ts/lib/Either";
 import * as O from "fp-ts/lib/Option";
-import * as TE from "fp-ts/lib/TaskEither";
 import * as T from "fp-ts/lib/Task";
+import * as TE from "fp-ts/lib/TaskEither";
```

Comment:
I would revert this change...  what do you think? For the sake of "changes minimization principle" 😉

Reply:
Reverted in https://github.com/pagopa/io-app/pull/4784/commits/06eee14b2d05d343daa3ae5bde40223044e118e5 👌🏻